### ISSUE TITLE
license is not written

### ISSUE THREAD

**Issue by sleep-walker** [jd/google-maps.el#5]
It seems that code has no visible license.

**Comment by jd** [comment]
The headers seem to indicate it's GPL, as GNU Emacs.

**Comment by sleep-walker** [comment]
I'm not a lawyer nor native speaker but I can see only information about license of GNU Emacs. Just close this issue if you disagree.

**Comment by apteryks** [comment]
Agreed that the license header could be clarified to mention this package's name (google-maps.el ?) rather than GNU Emacs, as this is misleading. A LICENSE file put at the root of the project would also help.
